src/main/java/com/exadel/discount/entity/Discount.java

```diff
@@ -1,46 +1,61 @@
 package com.exadel.discount.entity;
 
-import lombok.AllArgsConstructor;
-import lombok.Data;
-import lombok.EqualsAndHashCode;
-import lombok.NoArgsConstructor;
-import lombok.ToString;
+import lombok.*;
 import org.hibernate.annotations.GenericGenerator;
 
-import javax.persistence.Column;
 import javax.persistence.Entity;
-import javax.persistence.FetchType;
-import javax.persistence.GeneratedValue;
+import javax.persistence.Table;
 import javax.persistence.Id;
-import javax.persistence.JoinColumn;
-import javax.persistence.JoinTable;
+import javax.persistence.GeneratedValue;
+import javax.persistence.Column;
+import javax.persistence.ManyToOne;
 import javax.persistence.ManyToMany;
-import javax.persistence.Table;
+import javax.persistence.JoinTable;
+import javax.persistence.FetchType;
+import javax.persistence.JoinColumn;
+import java.time.LocalDateTime;
 import java.util.Set;
 import java.util.UUID;
 
-@Entity
-@Table(name = "discounts")
 @Data
 @NoArgsConstructor
-@AllArgsConstructor
-@EqualsAndHashCode(exclude = {"tags"})
-@ToString(exclude = {"tags"})
```

Comment:
Why did you delete this?

Reply:
maybe, to add this
@EqualsAndHashCode(exclude = {"tags", "coupons", "favorites"})
@ToString(exclude = {"tags", "coupons", "favorites"})

;)

Reply:
Added